sdk/samples/iot/README.md

```diff
@@ -578,33 +749,45 @@ The following samples use SAS authentication to connect to Azure IoT Hub or Azur
 
 ## Build and Run the Sample
 
-1. Compile the code:
+1. Build the Azure SDK for Embedded C directory structure.
 
-    - From the sdk root, create a build directory (eg `/build`). Change directory into your build directory.
-    - Build the directory structure and the samples:
+    From the sdk root directory `azure-sdk-for-c`:
+
+    Linux:
+
+    ```bash
+    mkdir build
+    cd build
+    cmake -DTRANSPORT_PAHO=ON ..
+    ```
+
+    Windows:
+
+    ```powershell
+    mkdir build
+    cd build
+    cmake -DTRANSPORT_PAHO=ON ..
+    ```
 
-      ```bash
-      cmake -DTRANSPORT_PAHO=ON ..
-      cmake --build .
-      ```
 
-2. From within the `build` directory, run the sample:
+2. Compile and run the sample from within the `build` directory.
 
     Linux:
 
     ```bash
+    make
```

Comment:
Not "cmake --build ." ?

Reply:
Both work.  I can use that if you prefer.